GlimpseImputationPipeline/Glimpse2Imputation.wdl

```diff
@@ -209,7 +205,11 @@ task GlimpsePhase {
             cmd="$cmd --checkpoint-file-in checkpoint.bin" 
         fi
 
-        eval $cmd
+
+
+        #check for read error which corresponds exactly to end of cram/bam block.  
+        #This currently triggers a warning message from htslib, but doesn't return any error
+        eval $cmd |& tee >(if grep -q "EOF marker is absent"; then echo "An input file appears to be corrupted.  Please fix before retrying."; exit 1; fi;) 
```

Comment:
Is it obvious how the input file would be corrupted? Like would it fail `samtools quickcheck` for example? I'm just trying to imagine coming across this error message, hoping to know what to do in that case

Reply:
http server returning an error that occurs at exactly the wrong time in the streaming of the cram.  I've never seen it, and I think it'd be super rare.  But I can't convince myself it's not possible.

I can adjust the message to be clearer and mention this possible cause as well.